portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -1,16 +1,23 @@
 package com.google.sps.servlets;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
 import java.io.IOException;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for deleting specified comment. */
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+/**
+ * Servlet responsible for deleting specified comment.
+ * 
+ * @author lucyqu
+ *
+ */
 @WebServlet("/delete-comment")
 public class DeleteCommentServlet extends HttpServlet {
```

Comment:
[OPTIONAL] From an actual information-security perspective, a servlet like this would want to check that the request to delete a comment posted by a given user is coming from that same user, since somebody could arbitrarily hit this endpoint without your delete button.